### PR TITLE
Lzma continued

### PR DESCRIPTION
**Pull request by Vertexwahn**
@Attempt3035

This tries to bring https://github.com/bazelbuild/bazel-central-registry/pull/1465 to an end

Either the changes from this branch are "copied-over" or this PR gets merged...

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (xz) have been updated in this PR. Please review the changes.

File: modules/xz/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://tukaani.org/",
+    "maintainers": [
+        {
```
Comment (meteorcloudy) [inline modules/xz/metadata.json:4]: @Attempt3035 Can you please confirm you are fine with being a maintainer for this module?

Comment (Attempt3035) [reply modules/xz/metadata.json:4]: Yep, happy to be maintainer!

Comment (meteorcloudy) [reply modules/xz/metadata.json:4]: Thanks!

**Comment by meteorcloudy** [comment]
Can you take a look at the presubmit failure?
```
(09:15:15) ERROR: Error computing the main repository mapping: in module dependency chain <root> -> xz@5.4.5: the MODULE.bazel file of xz@5.4.5 declares a different version (5.4.5.bcr.1)
```

**Comment by Vertexwahn** [comment]
@Attempt3035 Create a PR from your branch - I could not merge your PR in a trivial way since it had merge conflicts - sorry for this

**Comment by Vertexwahn** [comment]
@Attempt3035 reopened a new PR here -> https://github.com/bazelbuild/bazel-central-registry/pull/1538

**Comment by Vertexwahn** [comment]
@Attempt3035 Remove the `b/MODULE.bazel.lock` from you patch file!

**Pull request closed without merging**